bridge/elrond/client.go

```diff
@@ -324,13 +346,20 @@ func getStatusFromBuff(buff []byte) (byte, error) {
 }
 
 // GetActionIdForSetStatusOnPendingTransfer returns the action ID for setting the status on the pending transfer batch
-func (c *client) GetActionIdForSetStatusOnPendingTransfer(_ context.Context, batch *bridge.Batch) bridge.ActionId {
+func (c *client) GetActionIdForSetStatusOnPendingTransfer(ctx context.Context, batch *bridge.Batch) bridge.ActionId {
 	valueRequest := newValueBuilder(c.bridgeAddress, c.address.AddressAsBech32String(), c.log).
 		Func("getActionIdForSetCurrentTransactionBatchStatus").
 		BatchId(batch.Id)
 
-	for _, tx := range batch.Transactions {
-		valueRequest = valueRequest.BigInt(big.NewInt(int64(tx.Status)))
+	newBatch, err := c.GetPending(ctx)
+	if err != nil {
+		c.log.Error("Elrond: get pending batch failed in WasProposedSetStatus", "error", err.Error())
```

Comment:
optional: `err` instead of `err.Error()`

Reply:
done